tests/bdd/steps/h_api.py

```diff
@@ -14,10 +14,13 @@ def do_setup(self):
 
         httpretty.reset()
 
+        # Start interception
+        httpretty.enable()
+
```

Comment:
I'm not sure if this has the same effect or not, but registering after the enable seems sensible.